test/Npgsql.Tests/SystemTransactionTests.cs

```diff
@@ -334,6 +334,34 @@ public void BreakConnectorWhileInTransactionScopeWithCommit([Values] bool poolin
             Assert.That(ex.InnerException!.InnerException!.Message, Is.EqualTo(nameof(BreakConnectorWhileInTransactionScopeWithCommit)));
         }
 
+        [Test]
+        [IssueLink("https://github.com/npgsql/npgsql/issues/4085")]
+        public void OpenConnectionWithEnlistAndAbortedTransactionScope()
```

Comment:
```suggestion
        public void Open_connection_with_enlist_and_aborted_TransactionScope()
```

Reply:
Didn't name it like this so is looks similar to other tests, but sure.

Reply:
Yeah, I'll do a pass at some point to rename all the tests to snake_care. As you want.